torch/nn/functional.py

```diff
@@ -1139,7 +1139,7 @@ def rrelu(input, lower=1. / 8, upper=1. / 3, training=False, inplace=False):
 logsigmoid = _add_docstr(torch._C._nn.log_sigmoid, r"""
 logsigmoid(input) -> Tensor
 
-Applies element-wise :math:`\text{LogSigmoid}(x) = \log \left(\frac{1}{1 + \exp(-x_i)}\right)`
+Applies element-wise :math:`\text{LogSigmoid}(x) = \log \left(\frac{1}{1 + \exp(-x)}\right)`
```

Comment:
Could you instead change the LHS to be `x_i`?

Reply:
I have fixed it.